<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBuilderFieldsWidget</class>
 <widget class="QWidget" name="QueryBuilderFieldsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="btnLeftSelectItem">
         <property name="text">
          <string> &gt; </string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/chevron-right.svg</normaloff>:/img/chevron-right.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnLeftSelectItems">
         <property name="text">
          <string>&gt;&gt;</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/chevron-double-right.svg</normaloff>:/img/chevron-double-right.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRemoveItem">
         <property name="text">
          <string> &lt; </string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/chevron-left.svg</normaloff>:/img/chevron-left.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="btnRemoveItems">
         <property name="text">
          <string>&lt;&lt;</string>
         </property>
         <property name="icon">
          <iconset resource="resurses.qrc">
           <normaloff>:/img/chevron-double-left.svg</normaloff>:/img/chevron-double-left.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="btnAdd">
           <property name="toolTip">
            <string>Добавить</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resurses.qrc">
             <normaloff>:/img/itemAdd.png</normaloff>:/img/itemAdd.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnEdit">
           <property name="statusTip">
            <string>Изменить текущий элемент ( F2)</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resurses.qrc">
             <normaloff>:/img/itemEdit.png</normaloff>:/img/itemEdit.png</iconset>
           </property>
           <property name="shortcut">
            <string>F2</string>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnDelete">
           <property name="toolTip">
            <string>Удалить текущий (Alt+Shift+Del)</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="resurses.qrc">
             <normaloff>:/img/delete.png</normaloff>:/img/delete.png</iconset>
           </property>
           <property name="autoRaise">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resurses.qrc"/>
 </resources>
 <connections/>
</ui>
